dhis-2/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/controller/TrackerTrackedEntitiesExportControllerTest.java

```diff
@@ -475,29 +629,30 @@ private RelationshipType relationshipType( RelationshipEntity from, Relationship
         return type;
     }
 
-    private Relationship relationship( TrackedEntityInstance from, TrackedEntityInstance to )
+    private Relationship teiToTeiRelationship( TrackedEntityInstance from, TrackedEntityInstance to )
```

Comment:
It was intentional to write `relationship` as its clear what type `from` and `to` are. I think the method name changes make the tests less readable. Or was there some limitation that required this change 🤔 ?